assets/js/components/CookieOverlay.js

```diff
@@ -23,8 +24,16 @@ module.exports = class CookieOverlay {
     this.doc.querySelector('body').appendChild(this.$overlay);
   }
 
-  static previouslyAccepted(doc) {
-    return !!utils.getCookieValue('cookieNotificationAccepted', doc.cookie);
+  previouslyAccepted() {
+    return !!utils.getCookieValue('cookieNotificationAccepted', this.doc.cookie);
+  }
+
+  accept() {
+    this.$overlay.parentNode.removeChild(this.$overlay);
+    const expiryDate = 'Tue, 19 January 2038 03:14:07 UTC';
+    const domain = this.$elm.dataset.domain ? this.$elm.dataset.domain : 'elifesciences.org';
```

Comment:
Could extract from `this.window.location.hostname` rather than making it configuration?

Reply:
Alternatively we have been setting configuration on `window.elifeConfig`.

Reply:
@davidcmoulton in a recent stand-up was more favorable to "inject from the server-side" than to "detect with JavaScript"; can also change to `window.elifeConfig`

Reply:
Solved in-person: `window.elifeConfig` is the way to go